src/app/app.module.ts

```diff
@@ -3,10 +3,13 @@ import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
 import { LandingPageModule } from './landing-page/landing-page.module';
+import { PhoneComponent } from './phone/phone.component';
+import { AppRoutingModule } from './app-routing.module';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, LandingPageModule],
+  declarations: [AppComponent, PhoneComponent],
```

Comment:
It's better to create a PhoneModule and put it in imports rather than a component in declarations

Reply:
Done